include/drivers/yen/v6ksp_driver.h

```diff
@@ -0,0 +1,78 @@
+/*PGR-GNU*****************************************************************
+File: v6ksp_driver.h
```

Comment:
@cvvergara ma'am here I have created a new driver.h and remove the changes from the old driver.h.